FIAT/functional.py

```diff
@@ -71,39 +71,18 @@ def __init__(self, ref_el, target_shape,
         return
 
     def evaluate(self, f):
-        """Evaluates the functional on some callable object f."""
-        result = 0
-
-        # non-derivative part
```

Comment:
This is relevant for H(div) interpolation as well, let's discuss this some time.